src/main/java/de/espend/idea/php/annotation/doctrine/intention/DoctrineOrmRepositoryIntention.java

```diff
@@ -184,6 +180,6 @@ public String getFamilyName() {
     @NotNull
     @Override
     public String getText() {
-        return "Add Doctrine Repository";
+        return "Add doctrine repository";
```

Comment:
its still called "Doctrine" ;)

Reply:
Amended the original commit - thx for having a look.